### PR TITLE
Updated the android.tools version.

### PR DESCRIPTION
**Pull request by ritik2017**
Fixes Updated the tools version 

Changes: Updated tools version from 3.1.4(Older version) to 3.2.1(More recent version)

Screenshots/GIF for the change: [If possible, please add relevant screenshots/GIF]

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath "com.android.tools.build:gradle:3.1.4"
+    classpath 'com.android.tools.build:gradle:3.2.1'
```
Comment (abdulwd) [inline build.gradle]: In this file also update the `androidGradlePluginVersion`.

File: app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java [hunk 2]
```diff
@@ -150,8 +150,12 @@ public boolean onQueryTextSubmit(String s) {
       @Override
       public boolean onQueryTextChange(String s) {
         if (s.equals("")) {
+          View item = findViewById(R.id.menu_searchintext);
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java]: I don't think these changes are relevant to this issue.

Comment (ritik2017) [reply app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java]: Sorry,
Reverting these changes here.I have created another pull request regarding the issue.

File: build.gradle (hunk 1 above)
Comment (soloturn) [reply build.gradle]: @ritik2017 , you could squash commits and push force them so the history only shows the relevant change, one single commit changing the version in 2 places, a 2 line change. now it shows the commit of the search page change, and the revert, 6 commits in total.

**Comment by abdulwd** [comment]
@ritik2017 You should also update the gradle plugin version to `3.3`.

**Comment by abdulwd** [comment]
Also, update the gradle version to `4.10` here https://github.com/kiwix/kiwix-android/blob/7a3d5f8471b85badb8dd99bc21bb734d41a11e3b/gradle/wrapper/gradle-wrapper.properties#L6 and here https://github.com/kiwix/kiwix-android/blob/7a3d5f8471b85badb8dd99bc21bb734d41a11e3b/app/gradle/wrapper/gradle-wrapper.properties#L6

**Comment by ritik2017** [comment]
Thanks for the review @abdulwd

File: build.gradle (hunk 1 above)
Comment (ritik2017) [reply build.gradle]: Okay doing that here

**Comment by abdulwd** [comment]
@ritik2017 I have updated the versions in #925 to fix the build. Sorry for the inconvenience. You can take #563.

**Comment by abdulwd** [comment]
@soloturn Most probably, someone had imported the `app` module and added the Gradle wrapper to it by mistake. The Gradle wrapper in the `app` module is outdated and I have removed that in #925 and have updated the project's Gradle wrapper.

**Pull request closed without merging**